.github/workflows/ci.yml

```diff
@@ -32,43 +40,23 @@ jobs:
               run: |
                   brew install gtk+3
 
-            # check windows deps
-            - name: (Windows) Check available Gtk version
-              if: matrix.os == 'windows-latest'
-              shell: pwsh
-              run: |
-                  vcpkg search gtk | Select-String "^gtk " > gtk-version.txt
-                  Get-Content gtk-version.txt
-
-            # cache windows deps
-            - name: (Windows) Cache vcpkg packages
-              if: matrix.os == 'windows-latest'
-              id: cache-vcpkg
-              uses: actions/cache@v1
-              with:
-                  path: C:\vcpkg\installed
-                  key: ${{ runner.os }}-vcpkg-gtk-${{ hashFiles('gtk-version.txt') }}
-
             # windows deps
             - name: (Windows) Install Gtk build environment
-              if: matrix.os == 'windows-latest' && steps.cache-vcpkg.outputs.cache-hit != 'true'
+              if: matrix.os == 'windows-latest'
+              shell: msys2 {0}
               run: |
-                  vcpkg install gtk:x64-windows
-                  copy C:\vcpkg\installed\x64-windows\lib\gtk-3.0.lib C:\vcpkg\installed\x64-windows\lib\gtk-3.lib
-                  copy C:\vcpkg\installed\x64-windows\lib\gdk-3.0.lib C:\vcpkg\installed\x64-windows\lib\gdk-3.lib
+                pacman -S --noconfirm mingw-w64-x86_64-gtk3 mingw-w64-x86_64-toolchain
```

Comment:
If you add these to field `install` of `setup-msy2, packages will be cached for faster setup. It's done automatically.

```yml
            - uses: msys2/setup-msys2@v2
              with:
                path-type: inherit
                install: >
                  mingw-w64-x86_64-gtk3
                  mingw-w64-x86_64-toolchain
              if: matrix.os == 'windows-latest'
```

Reply:
I assume that's *instead* of the pacman command? Thanks, I'll experiment.

Reply:
Yes. In fact, the action will just execute pacman internally: https://github.com/msys2/setup-msys2/blob/master/main.js#L266. It's a different order, but the same result.